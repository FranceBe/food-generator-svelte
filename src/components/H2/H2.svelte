<script>
    export let title
</script>
<style>
    h2 {
        font-size: 1.25rem;
        text-align: center;
        text-shadow: 1px 1px #EBA789;
    }
 </style>

<h2>{title}</h2>
