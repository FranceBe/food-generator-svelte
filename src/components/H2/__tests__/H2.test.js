import { cleanup, render } from '@testing-library/svelte'
import H2 from 'components/H2/H2.svelte'

describe('H2', () => {
  afterEach(() => {
    cleanup()
  })
  it('should match snapshot', () => {
    const { container } = render(H2, {
      props: {
        title: 'My title',
      }
    })
    expect(container).toMatchSnapshot()
  })
  it('should render an h2 tag', () => {
    const { container, getAllByText } = render(H2, {
      props: {
        title: 'My title',
      }
    })
    expect(container.getElementsByTagName('h2')).toHaveLength(1)
    expect(getAllByText('My title').length).toBe(1)
  })
})
