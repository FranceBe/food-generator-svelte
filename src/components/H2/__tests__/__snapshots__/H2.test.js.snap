// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`H2 should match snapshot 1`] = `
<body>
  <div>
    <h2>
      My title
    </h2>
  </div>
</body>
`;
